test/api-test.js

```diff
@@ -435,13 +435,19 @@ describe('RippleAPI', function () {
           "maxLedgerVersion": 8820051
         }
       }
-
+      
       return this.api.preparePayment(
         address, requests.preparePayment.normal, localInstructions).then(
           _.partial(checkResult, expectedResponse, 'prepare'));
     });
   });
 
+  it('createPaymentTransaction - normal', function(){
+    return this.api.createPaymentTransaction(address, requests.preparePayment.normal).then((res)=>{
```

Comment:
`createPaymentTransaction` should not return a promise, so `then` should not be used here.